[2024-05-14T05:29:38.629+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-14T05:29:38.824+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: tugas_workflow_v1.task_3 scheduled__2024-01-02T00:00:00+00:00 [queued]>
[2024-05-14T05:29:38.899+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: tugas_workflow_v1.task_3 scheduled__2024-01-02T00:00:00+00:00 [queued]>
[2024-05-14T05:29:38.902+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 6
[2024-05-14T05:29:39.017+0000] {taskinstance.py:2330} INFO - Executing <Task(BashOperator): task_3> on 2024-01-02 00:00:00+00:00
[2024-05-14T05:29:39.038+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=3891) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-05-14T05:29:39.041+0000] {standard_task_runner.py:63} INFO - Started process 3942 to run task
[2024-05-14T05:29:39.048+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'tugas_workflow_v1', 'task_3', 'scheduled__2024-01-02T00:00:00+00:00', '--job-id', '748', '--raw', '--subdir', 'DAGS_FOLDER/__pycache__/prioritas_1.py', '--cfg-path', '/tmp/tmpq705r4by']
[2024-05-14T05:29:39.053+0000] {standard_task_runner.py:91} INFO - Job 748: Subtask task_3
[2024-05-14T05:29:39.323+0000] {task_command.py:426} INFO - Running <TaskInstance: tugas_workflow_v1.task_3 scheduled__2024-01-02T00:00:00+00:00 [running]> on host 0391a550332e
[2024-05-14T05:29:39.994+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='sinta' AIRFLOW_CTX_DAG_ID='tugas_workflow_v1' AIRFLOW_CTX_TASK_ID='task_3' AIRFLOW_CTX_EXECUTION_DATE='2024-01-02T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-01-02T00:00:00+00:00'
[2024-05-14T05:29:40.013+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-14T05:29:40.029+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-05-14T05:29:40.059+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'mkdir -p our_works']
[2024-05-14T05:29:40.230+0000] {subprocess.py:86} INFO - Output:
[2024-05-14T05:29:40.256+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-05-14T05:29:40.263+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-14T05:29:40.351+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=tugas_workflow_v1, task_id=task_3, run_id=scheduled__2024-01-02T00:00:00+00:00, execution_date=20240102T000000, start_date=20240514T052938, end_date=20240514T052940
[2024-05-14T05:29:40.722+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-05-14T05:29:40.900+0000] {taskinstance.py:3498} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-05-14T05:29:40.911+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
